code/datums/weather/weather.dm

```diff
@@ -105,14 +105,16 @@
 	weather_duration = rand(weather_duration_lower, weather_duration_upper)
 	START_PROCESSING(SSweather, src)
 	update_areas()
-	for(var/M in GLOB.player_list)
+	addtimer(CALLBACK(src, .proc/start), telegraph_duration)
```

Comment:
Why move this

Reply:
I moved it before the check SOUND_WEATHER check in case someone turns their weather noises back on between the start and the middle.

Reply:
You dont need this

Reply:
Are you sure you really need this?